### ISSUE TITLE
nvda not reading disk check dialog window when getting to window with alt+tab

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14244]
### Steps to reproduce:
- open this computer window
- open the properties of some disk
- go to tools tab
- access the check now button
- the dialogue error checking should be open.
- now press `alt + tab` and put focus on another window
- now press `alt+tab` again until disk check window.

### Actual behavior:
NVDA reports the name of the opened window wrongly, it reports the window name as properties of the open window below the current window.
Despite announcing properties when walking with `tab` nvda announces unknown and doesn't read anything.

log: [disc.txt](https://github.com/nvaccess/nvda/files/9770180/disc.txt)

### Expected behavior:
When reaching the window with `alt + tab`
NVDA should read the correct window name as disk check.
When walking with tab should correctly read the disk check window.

When NVDA doesn't read the name of the open disk check window if you press `NVDA + n` to open the NVDA menu and press `esc` to vote for the disk check window everything works fine.

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
`nvda.exe, NVDA alpha-26777,f4955df6`

#### Windows version:
`11 22623.741`

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
This works as expected in NVDA 2025.1 beta and Windows 11 24 H2.
Disk check indow is a nested dialog under disk properties window, so if you pres and hold alt+tab, NVDA reports "sidk properties". When you release the alt+tab keystroke, NVDA reports "disk check" and the descriptive text that belongs to the disk check dialog.
Pressing tab afterwards navigates the controls of the dialog as expected, without saying "unknown" or something similar.
Closing as works for me.
If you are still having this issue, please comment and we can reopen.
